index.js

```diff
@@ -107,15 +107,24 @@ class UpdateNotifier {
 			};
 		});
 	}
+	isYarnGlobal() {
+		const isWindows = process.platform === 'win32' || process.env.OSTYPE === 'cygwin' || process.env.OSTYPE === 'msys';
```

Comment:
Should only be `const isWindows = process.platform === 'win32';`

Reply:
👌